### DISCUSSION TITLE
Task failed to start

### DISCUSSION THREAD

**Discussion by joshpollara** [bottlerocket-os/bottlerocket#1786] [General]
Hello,

I'm trying to move from the latest ECS optimized Amazon Linux 2 to Bottlerocket. When I update my auto-scaling group and bring up an instance with Bottlerocket my ECS tasks fail to start. It looks like the tasks don't even try to schedule on the Bottlerocket container instance. Eventually the task stops with _Task failed to start_.

I'm having a difficult time troubleshooting due to the lack of logs from the host itself and ECS. Can anyone offer me any advice? Thank you.

Josh

**Comment by tjkirch** [comment]
Hi Josh!  One thing that would help us figure out the issue is knowing whether you've seen the [ECS quickstart guide](https://github.com/bottlerocket-os/bottlerocket/blob/develop/QUICKSTART-ECS.md).  It doesn't specifically discuss ASGs, but does describe the minimum user data you'll need, which is rather different than with Amazon Linux.  If you have seen it, could you share your user data and how the ASG is configured?  Thanks!
